pkgs/development/tools/omnisharp-roslyn/updater.sh

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p curl jq common-updater-scripts nuget-to-nix
+#! nix-shell -i bash -p curl jq common-updater-scripts nuget-to-nix xmlstarlet dotnet-sdk
```

Comment:
Adding `dotnet-sdk` here messes up with the update script as it needs .NET 7 for the restore

Reply:
> It actually seems to work for me with both the source and binary sdks. Could it have something to do with setting RuntimeFrameworkVersion? In any case, I think it can be removed.

No idea honestly, but if it worked for you then no issue keeping it that way then!